cloud-haven-react-app/src/components/ProfileDropdown/ProfileDropdown.js

```diff
@@ -0,0 +1,21 @@
+import React, { useState } from "react";
+import './ProfileDropdown.css';
+
+const ProfileDropdown = () => {
+
+    return (
+        <div class="dropdown">
+         <button class="btn btn-light dropdown-toggle" type="button" data-toggle="dropdown">
+            Username
+         <span class="caret"></span></button>
```

Comment:
```suggestion
        <div class="dropdown">
           <button class="btn btn-light dropdown-toggle" type="button" data-toggle="dropdown">
              Username
              <span class="caret"/>
           </button>
```
Just aligning things to be a bit easier to read

Reply:
Unresolving this since I'm not sure this change has been made (github is showing it as not having changed)

Reply:
nvm Github was just not refreshing